.circleci/config.yml

```diff
@@ -208,7 +208,7 @@ jobs:
                       git rm -rf .
                       # Add the CNAME record for pxblue.eaton.com
                       # if [ $CIRCLE_BRANCH == $MASTER_BRANCH ]; then
-                      #   echo "pxblue-components.eaton.com/react" > CNAME
+                      #   echo "brightlayer-ui-components/react" > CNAME
```

Comment:
@jeffvg  @joebochill  Do you remember what this is doing?

Reply:
believe its related to this...
https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site